src/components/ProductCard/ProductCard.jsx

```diff
@@ -0,0 +1,14 @@
+import * as React from "react"
+import "./ProductCard.css"
+
+export default function Products(props) {
+  return (
+    <div className="product-card">
+      <p className="product-name">{props.name}</p>
+      <p className="product-quantity">{props.quantity}</p>
+      <button className="product-button" onClick={() => props.handleProductQuantity(props.name, "add")}>+</button>
```

Comment:
For handling these action types, raw strings can be dangerous/hard to maintain. It's possible another developer wants to extend this component, add new operations, or just cant remember if it was "minus" or "subtract". 

There are 2 options I would consider to address this situation (not sure what the most JS way to do this is) but:
1. Create different functions e.g. `addProductQuantity`, `decreaseProductQuantity`, etc.  These can just call this function with the appropriate argument. 
2. Either in addition to option 1 or as an alternative, you could use an [`enum`](https://www.sohamkamani.com/javascript/enums/#when-to-use-enums-in-javascript)  that contains all of the available operations. This makes it more clear what options are available. In a typed language like typescript, this would also let the linter (or compiler for compiled languages) check that the provided option is valid in all cases without even running the code.

Reply:
I created an enum like this:
`const Operations = Object.freeze({
  Add: Symbol("add"),
  Subtract: Symbol("subtract"),
  Erase: Symbol("erase")
})`
And used this as parameters